src/arb/ArbTypes.jl

```diff
@@ -35,6 +35,17 @@ mutable struct arf_struct
   size::UInt # mp_size_t
   d1::UInt # mantissa_struct
   d2::UInt
+
```

Comment:
Please add back the default constructor. It is used upstream.

Reply:
ok, bad hecke. By default constructor you mean ... ?

Reply:
Apparently we are using `arf_struct(::Int64, ::Int64, ::Int64, ::Int64)`.

Reply:
I'm sorry, I had to look:
LinearAlgebra/LatEnum.jl is only using it C-style to avoid attaching a destructor
Misc/Arb.jl is throwing between init and clear thus possibly leaking
Misc/NumFieldOrd/NfOrd/Zeta.jl is throwing between init and clear thus possibly leaking
QuadForm/MassQuad.jl is using it without init which is currently OK